### DISCUSSION TITLE
Using Kryptor keys in mails?

### DISCUSSION THREAD

**Discussion by dawidmachon** [samuel-lucas6/Kryptor#19] [Q&A] [answered]
Is the generated key pair in Kryptor sufficient to encrypt email communication as in gnupg?

**Comment by samuel-lucas6** [comment, marked as answer]
I'm afraid not since the key format hasn't been standardised like OpenPGP has. You can read more about the key format [here](https://www.kryptor.co.uk/technical-details#generating-key-pairs).

**Comment by covert-encryption** [comment]
Email is a fundamentally insecure platform. People have spent 25 years trying to secure it and are still failing. Please don't try to secure email but rather switch to something better. https://latacora.micro.blog/2019/07/16/the-pgp-problem.html
